### ISSUE TITLE
What is the maximum waiting time when retrying?

### ISSUE THREAD

**Issue by piotr-s-brainhub** [renovatebot/renovate#3729]
What is the maximum time to update a PR after checking `If you want to rebase/retry this PR, check this box` on an open PR?

What is the maximum time to open a new PR after checking `If you want to rebase/retry this PR, check this box` on a merged PR?

**Comment by rarkins** [comment]
Renovate should run within maximum 30 seconds in most cases. Only exception is when it’s upgrading itself as then it could take a few minutes. 

BTW, GitHub’s checkboxes are big very compatible with Firefox browser. For some reason it caches too aggressively and does not show the correct state sometimes.

**Comment by piotr-s-brainhub** [comment]
Today I had a situation that I wanted renovate to open a PR in a given repo ASAP so I checked this checkbox but it opened after about 20 minutes together with the other repos.

**Comment by rarkins** [comment]
One other possibility is that if there's a job already in progress for that repo when you tick the box then the new job you request cannot start until the old one finishes. If it happens again, please take a note of the times and email me the details so I can do a detailed check of the webhook and job logs.

**Comment by piotr-s-brainhub** [comment]
Ok, thanks.
